### ISSUE TITLE
Replace CMIAPI with a more marketable name

### ISSUE THREAD

**Issue by matthewGEA** [cesmii/API#7]
FROM  CMIAPI -- Contextualized Manufacturing Information Application Programming Interface

TO:

- Industrial Information Interface (Triple-I)
- Industrial Information Contract (I2C) -- for you @aronsemle 
- Industrial Information Hydrant
- or other
![image](https://github.com/user-attachments/assets/1da00a30-2794-475f-bf71-0212b8f5b51b)

**Comment by aronsemle** [comment]
100% it needs a name that's easier to market/remember. I'll give this some thought as well.

**Comment by barnstee** [comment]
I3API is catchy

**Comment by jwise-mfg** [comment]
Or i4API -- for Industry 4.0?
I know its buzzwordy, but...

**Comment by barnstee** [comment]
with a capital I, yes I4API. Or SMAPI?

**Comment by slorkowski** [comment]
I like I4API..,.,but I4 might be over-used. 
Love SMAPI.. since it's a Smart Mfg API and is inline with other SM stuff , like Profiles, Interoperability Platform.
Like Ind Info Hydrant since it's very visual. However, Siemens has an IIH...Ind Info Hub and High Byte has an  Intelligence Hub.
Maybe I'll get more creative this weekend, after a beer :-) Have a great weekend!

**Comment by scottmwyant** [comment]
I wish I had something concrete to offer here; for now I'll share a simple observation.

A short and sweet acronym is not really possible if you end it with "API".  Perhaps we could settle on something that simply uses "A" to represent the API part, whether it appears first or last. Kind of like "OPC" originally referring to "OLE for Process Control".

**Comment by matthewGEA** [comment]
Perhaps a recursive acronym:

**Qsie** = Qsie standardizes information exchange

**Comment by aronsemle** [comment]
I agree with Scott that keeping API in the name makes it hard to be "catchy". Here are some ideas I came up. Having the word "Smart" in it could be interesting given the CESMI origin.

Industrial Data Framework (IDF)
Industrial Data Fabric (IDF)    // AWS has this already
Industrial Data Standard (IDS)
Manufacturing Data Exchange (MDX)
Manufacturing Data Framework (MDF) 
Manufacturing Data Foundation (MDF)
Smart Manufacturing Data Foundation (SMDF)
Manufacturing Data Platform (MDP)
Industrial Systems Exchange (ISE)
Smart Operations Framework (SOF)
Smart Industrial Framework (SIF)
Operational Intelligence Framework (OIF)
Operational Technology Framework (OIF)

**Comment by slorkowski** [comment]
I like those, Aron, and to piggy back off your thoughts...
Smart Industrial System Exchange (SMISE since SISE is used by Cisco)
Smart Industrial System Interoperability (SISI or SMISI)
Smart Manufacturing Software Interoperability (SMSI)

**Comment by jwise-mfg** [comment]
The group consensus was I3X for "Industrial Information Interface eXchange"

**Comment by matthewGEA** [comment]
Purpose a further improvement: 

FROM "Industrial Information Interface Exchange (I3X)"

TO: "Industrial Information Exchange (I2X)" 

The existing name reads as if "interfaces" are being exchanged, rather than information being exchanged.

**Comment by jwise-mfg** [comment]
Let's not [bikeshed](https://effectiviology.com/bikeshedding-law-of-triviality/) on the name. This issue was closed with a consensus vote by the WG, let's move on to more important issues -- unless a critical blocker comes up (eg: existing trademark, etc...)

**Comment by jwise-mfg** [comment]
Btw, with this expansion, the acronym works: Industry, Interoperability, and Information eXchange

**Comment by matthewGEA** [comment]
Another?
Industrial Information for Interoperability eXchange
